app/components/Form/ApplicationForm.tsx

```diff
@@ -186,8 +186,17 @@ const ApplicationForm: React.FC<Props> = ({
   } = application;
 
   const formErrorSchema = useMemo(() => validate(jsonData), [jsonData]);
-  const sectionName = getSectionNameFromPageNumber(pageNumber);
-
+  const filteredUiSchemaOrder = uiSchema['ui:order'].filter((formName) => {
```

Comment:
Nice catch, that I missed when I did the spike. Though it took me a minute to figure out what is happening here, a comment explaining might be helpful to future generations 😅